app/client/components/form/SelectField.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import {Field} from 'redux-form'
+
+import {WrappedInput} from './WrappedInput'
+import styles from "./styles.module.css";
+import FormValidation from "./FormValidation";
+
+interface SelectFieldJson {
```

Comment:
Are we exclusively using TypeScript now?

Reply:
i'd say incrementally get into it. if you're working on an existing file that'll be hard to convert, don't bother or wait til you get more experience in it :) but if you're doing something simple from scratch, would be worth giving it a shot. the more typescript the safer the code